cpp/ps0003/cpp_ps0003.cpp

```diff
@@ -36,4 +23,7 @@ int main() {
   // ternary condition-----
   std::string result = (number % 2 == 0) ? "Number is even" : "Number is odd";
   std::cout << result << std::endl;

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console